new-pipeline/src/dist.js

```diff
@@ -120,9 +120,9 @@ $(function() { Telemetry.init(function() {
             return getAggregate(entry2.histograms) - getAggregate(entry1.histograms);
           });
           
+          var keys = gCurrentHistogramsList.map(function(entry) { return entry.title; });
```

Comment:
arrow functions are coming to Chorme 45 (currently supported in Chrome Beta, launching in a week) and Edge supports them already: https://www.chromestatus.com/feature/5047308127305728

I'm tempted to ask you to switch to arrow functions in advance.. it reads a lot cleaner